nodebuilder/share/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	EDSStoreParams *store.Parameters
 
 	UseShareExchange bool
+	UseShareSwap     bool
```

Comment:
Better keep it Shwap and rename the above one ot shrex, so there is less confusions around namings